csrc/multidevice/utils.h

```diff
@@ -21,12 +22,43 @@ NVF_API bool distributedEnabled();
 // Checks that the other non-reduction axis are not parallelized on Didx
 NVF_API bool isSharded(TensorView*);
 
-// Returns the subset of tvs which elements have the same multi-device sharding
-// as ref
+// Returns the subset of tvs which elements have the different multi-device
+// sharding as ref
 template <typename TvIterator>
 std::unordered_set<TensorView*> getTvsWithDifferentSharding(
     TensorView* ref,
-    TvIterator tvs);
+    TvIterator tvs) {
+  std::unordered_set<TensorView*> ret;
```

Comment:
Is this just a move or should I also review the implementation?

Reply:
Just a move!